packages/3d_image_encoders/src/3d_image_encoders/i3d/run_training_on_covid_dataset.py

```diff
@@ -4,7 +4,7 @@
 import torchvision
 
 from epsdatasets.helpers.covid.covid_dataset_helper import CovidDatasetHelper
```

Comment:
I removed 'helpers' subdir when migrating to epsml. Can you fix in all the other places as well?

```suggestion
from epsdatasets.covid.covid_dataset_helper import CovidDatasetHelper

Reply:
This is done. Please see commit [65c70ea](https://github.com/EpsilonLabsInc/epsml/pull/2/commits/65c70eac7ea0ebfb5498f5a902f462684189ab03)